etcdctl/ctlv3/command/printer.go

```diff
@@ -20,7 +20,7 @@ import (
 	"strings"
 
 	v3 "github.com/coreos/etcd/clientv3"
-	"github.com/dustin/go-humanize"
+	humanize "github.com/coreos/etcd/gopath/src/github.com/dustin/go-humanize"
```

Comment:
Let's keep import path as it is? `"github.com/dustin/go-humanize"`

Reply:
That's odd. I didn't make that change. Will change it back.